src/core/gosling.schema.ts

```diff
@@ -403,8 +403,10 @@ export interface MatrixData {
     url: string;
 }
 
+// !!! Transformation is applied in the same order (i.e., filter, log, and then stack)
```

Comment:
I think we need to make this more flexible (e.g., allow users to determine the order of data transformation) in the future.